doc/classes/String.xml

```diff
@@ -313,6 +313,24 @@
 				[b]Note:[/b] Strings with equal hash values are [i]not[/i] guaranteed to be the same, as a result of hash collisions. On the countrary, strings with different hash values are guaranteed to be different.
 			</description>
 		</method>
+		<method name="hex_decode" qualifiers="const">
+			<return type="PackedByteArray"/>
+			<description>
+				Decodes a hexadecimal string as a [PackedByteArray].
+				[codeblocks]
+				[gdscript]
+				var text = "hello world"
+				var encoded = text.to_ascii_buffer().hex_encode()
+				print(buf.hex_decode().get_string_from_ascii())
```

Comment:
It might be better to use `to_utf8_buffer()` in the example so that it applies to other languages too.